tests/test_routes.py

```diff
@@ -6,3 +9,54 @@ def test_index_page(flask_test_client):
 def test_upload_page(flask_test_client):
     response = flask_test_client.get("/upload")
     assert response.status_code == 200
+
+
+def test_upload_xlsx_successful(
+    requests_mock, example_pre_ingest_data_file, flask_test_client
+):
+    requests_mock.post(
+        "http://localhost:8080/ingest",
```

Comment:
To make this clearer, can we make the url in the `unit_test` environment. `http://data-store`?